lighthouse-core/scripts/lantern/download-traces.sh

```diff
@@ -29,7 +29,7 @@ rm -rf lantern-data/
 mkdir -p lantern-data/ && cd lantern-data
 echo $VERSION > version
 
-curl -o lantern-traces.tar.gz -L https://storage.googleapis.com/lh-lantern-data/lantern-traces-$VERSION.tar.gz
+curl -o golden-lantern-traces.zip -L https://storage.googleapis.com/lh-lantern-data/golden-lantern-traces-$VERSION.zip
```

Comment:
renamed file b/c we are also keeping the big zip files in this bucket, and I wanted to reuse the naming that the lantern scripts use.